lib/Apache/Session/Browseable/Redis.pm

```diff
@@ -98,7 +98,7 @@ sub get_key_from_all_sessions {
         eval {
             my $v   = $redisObj->get($k);
             next unless $v;
-            my $tmp = unserialize($v);
+            my $tmp = eval { unserialize($v) };
```

Comment:
This call is already inside an eval. I don't see how it helps and I can't reproduce the failure mentionned in #40